src/userlist/userListActions.js

```diff
@@ -21,3 +20,9 @@ export const sendGetUsers = (auth: Auth) =>
       users: response,
     });
   };
+
+export const sendInitialGetUsers = (auth: Auth) =>
```

Comment:
Why name actions with the `send` prefix? I find it superfluous personally (and maybe a little confusing)